__config__/webpack/webpack.config.dev.js

```diff
@@ -1,114 +1,50 @@
-                    const src = require('../../__tools__/demo/src.server')
-                        .default;
-                    try {
-                        res.send(src(req.params[0]));
-                    } catch (e) {
-                        log(e);
-                    }
-                });
-
-                app.get('/pages/*', async (req, res) => {
-                    const page = require('../../src/server').default;
-                    const pageType = req.params[0].split('/pages/')[0];
-                    const data = require(`../../.data/${pageType}`);
-
-                    try {
-                        res.send(await page(data));
-                    } catch (e) {
-                        log(e);
-                    }
-                });
-
-                app.get('/', async (req, res) => {
-                    try {
-                        res.send(`
-                        <html>
-                        <h3>pages</h3>
-                        <ul>
-                        ${fs
-                            .readdirSync(
-                                path.resolve(__dirname, '../../src/pages'),
-                            )
-                            .map(page => {
-                                const name = page.replace(/.js$/, '');
-                                return `<li><a href="/pages/${name}">${name}</a></li>`;
-                            })
-                            .join('')}
-                        </ul>
-                        <h3>components</h3>
-                        <ul>
-                        ${fs
-                            .readdirSync(
-                                path.resolve(__dirname, '../../src/components'),
-                            )
-                            .filter(page => page.endsWith('.demo.js'))
-                            .map(page => {
-                                const name = page.replace(/.demo.js$/, '');
-                                return `<li><a href="/demo/components/${name}">${name}</a></li>`;
-                            })
-                            .join('')}
-                        </ul>
-                        </html>
-                        `);
-                    } catch (e) {
-                        log(e);
-                    }
-                });
-            },
         },
         plugins: [
-            new FriendlyErrorsWebpackPlugin({
-                compilationSuccessInfo: {
-                    messages: [
-                        `DEV server running at ${chalk.blue.underline(
-                            'http://localhost:3000',
-                        )}`,
-                    ],
-                },
-            }),
+            new webpack.HotModuleReplacementPlugin(),
+            new webpack.NamedModulesPlugin(),
+            friendlyErrorsWebpackPlugin,
         ],
     },
-    server: {},
+    server: {
+        name: 'server',
+        target: 'node',
```

Comment:
i think this is set already?

Reply:
i'm going to add a few "i think this is set already?" comments, sorry for the spam :)

Reply:
No worries, I'll do a cross check with the main config now and remove anything that's specified there from this config!